apps/trading/components/constants.ts

```diff
@@ -1,2 +1 @@
 export const THROTTLE_UPDATE_TIME = 500;
-export const ONBOARDING_VIEWED_KEY = 'vega_onboarding_viewed';
```

Comment:
on this key in localStorage bases e2e tests as they suppress onboarding popup

Reply:
Updated in both cypress. And here's a PR for console-test https://github.com/vegaprotocol/console-test/pull/120